src/core/annotation.js

```diff
@@ -3020,18 +3036,26 @@ class ChoiceWidgetAnnotation extends WidgetAnnotation {
       return super._getAppearance(evaluator, task, annotationStorage);
     }
 
-    if (!annotationStorage) {
+    if (!annotationStorage && this.appearance) {
```

Comment:
Won't this cause an error to be thrown below if `annotationStorage === undefined` *and* `this.appearance === undefined`?

Reply:
Yep, you're right, thanks for catching that.
I updated to have something similar to what we have in the appearance getter for text fields.